python/az/aro/azext_aro/custom.py

```diff
@@ -316,7 +316,7 @@ def get_cluster_network_resources(cli_ctx, oc):
     # Ensure that worker_profiles exists
     # it will not be returned if the cluster resources do not exist
     if oc.worker_profiles is not None:
-        worker_subnets = {w.subnet_id for w in oc.worker_profiles if w}
+        worker_subnets = {w.subnet_id for w in oc.worker_profiles}
```

Comment:
@bennerv is my change here OK with you?

Reply:
@jim-minter I would leave it in.  It's performing a [few checks](https://stackoverflow.com/questions/100732/why-is-if-not-someobj-better-than-if-someobj-none-in-python) on each worker_profile before getting the subnet id.   I don't believe there would be a scenario where one of the profiles was `None`, but this would prevent it anyway.